app/Models/Equipment/Breakage.php

```diff
@@ -57,4 +57,22 @@ public function user()
     {
         return $this->belongsTo('App\Models\Users\User');
     }
+
+    public function saveImage($image)
+    {
+        // save image to /resources/breakages
+        $filename = uniqid() . '.' . $image->extension();
+        $image->move(resource_path("breakages/"), $filename);
+        return $filename;
+    }
+
+    /**
+     * Define the foreign-key relationship.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function images()
+    {
+        return $this->hasMany('App\Models\Equipment\BreakageImage', 'report_id');
```

Comment:
Do we want to sort this by the position? Otherwise, we probably don't need to store the position at all.